client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -396,6 +398,7 @@ export class CheckoutThankYou extends React.Component {
 			delayedTransferPurchase = find( purchases, isDelayedDomainTransfer );
 			wasMarketplaceProduct = purchases.some( isMarketplaceProduct );
 			wasDomainMappingOnlyProduct = purchases.length === 1 && purchases.some( isDomainMapping );
+			wasTitanEmailPurchase = purchases.length === 1 && purchases.some( isTitanMail );
```

Comment:
This should be written as `wasTitanEmailProductOnly` since it describes the logic scenario better.

Reply:
Or maybe `wasTitanEmailOnlyProduct` to be consistent with `wasDomainMappingOnlyProduct`?